src/components/dashboard/FirstSteps/index.tsx

```diff
@@ -0,0 +1,77 @@
+import useBalances from '@/hooks/useBalances'
+import { useAppSelector } from '@/store'
+import { selectOutgoingTransactions } from '@/store/txHistorySlice'
+import { type ReactNode } from 'react'
+import { Card, WidgetBody, WidgetContainer } from '@/components/dashboard/styled'
+import { LinearProgress, List, ListItem, ListItemIcon, Typography } from '@mui/material'
+import CircleOutlinedIcon from '@mui/icons-material/CircleOutlined'
+import CheckCircleRoundedIcon from '@mui/icons-material/CheckCircleRounded'
+
+const StatusItem = ({ children, completed = false }: { children: ReactNode; completed?: boolean }) => {
+  return (
+    <ListItem disableGutters>
+      <ListItemIcon sx={{ minWidth: 24, mr: 1, color: (theme) => theme.palette.primary.main }}>
+        {completed ? <CheckCircleRoundedIcon /> : <CircleOutlinedIcon />}
+      </ListItemIcon>
+      {children}
+    </ListItem>
+  )
+}
+
+type StatusProgressItems = Array<{
+  name: string
+  completed: boolean
+}>
+
+const StatusProgress = ({ items }: { items: StatusProgressItems }) => {
+  const totalNumberOfItems = items.length
+  const completedItems = items.filter((item) => item.completed)
+  const progress = Math.round((completedItems.length / totalNumberOfItems) * 100)
```

Comment:
These three vars are only needed to compute the `progress` var. Extract them into a `calculateProgress` function?